### DISCUSSION TITLE
Why does winget not run from powershell but runs from powershell via windows terminal?

### DISCUSSION THREAD

**Discussion by Battler624** [microsoft/winget-cli#4427] [PowerShell]
As can be seen here.

https://github.com/microsoft/winget-cli/assets/9511158/6bb893f2-1fc0-4dc6-a896-2755fb5f3a9b

**Comment by Battler624** [comment]
Fresh windows 11 install. I dont recall this happening on Windows 10.

**Reply by denelon** [reply]
I don't think I've seen this behavior before. Is this the latest version after updating "App Installer" via the store?

**Reply by Battler624** [reply]
Yes

**Reply by denelon** [reply]
I created:
* https://github.com/microsoft/winget-cli/issues/4519

Can you share the output from `winget --info` on that issue so we can start to dig in?

**Reply by Battler624** [reply]
@denelon, posting now.
